scoala-online/main
# Feat/#26 create navbar

### This pull request solves issue:

Closes: #26 

### Which app is this issue related to?


### Types of changes


### Wiki Updates


### Testing checklist


#### Security Filters - **_For API Only_**

**_Please make sure to add the respective Security Filters before submitting a PR_**


#### Compatibility tests - **_For E-Learning React App Only_**


### What changes have been done within this issue?

I added the navigation menu for the UI.

### How should this be manually tested?

Launch the e-learning-app and the navbar will appear on top of the page. Clicking on 'Materiale'  should open the grade sidebar. Clicking on any grade should open the subject sidebar. Clicking outside the sidebars will close them. Resize the viewport to see how the elements scale or change when the sizes are too small.

### Screenshots or example usage
